docker.d/init_worker.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+# == START: this is what we need to configure ==
+test $SSH_USER
+test $SSH_KEY
+# == END:   this is what we need to configure ==
+
+case $ENV in
+  dev|stage|prod) ;;
+  *)
+    exit 1
+    ;;
+esac
+
+case $COT_PRODUCT in
+  firefox)
+    export TASKCLUSTER_SCOPE_PREFIX="project:releng:${PROJECT_NAME}script:"
+    ;;
+  thunderbird)
+    export TASKCLUSTER_SCOPE_PREFIX="project:comm:thunderbird:releng:${PROJECT_NAME}script:"
+    ;;
+  *)
+    exit 1
+    ;;
+esac
+
+export HG_SHARE_BASE_DIR=/tmp/share_base
+export SSH_KEY_PATH=$CONFIG_DIR/ssh_key_$SSH_USER
+
+echo $SSH_KEY | base64 -d > $SSH_KEY_PATH
```

Comment:
Same here, if we can unset this once we're done configuring, we may avoid leaking the secret.

Edit: I also found `GITHUB_OAUTH_TOKEN` below -- perhaps we should put all the secret env var names in a `remove_secrets_from_env_var.sh` that we can source before we launch scriptworker?

Reply:
:+1: i will unset all the variable at the end of the init.sg script.